### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#158] [Esercizi - Bertini] [answered]
![Screenshot 2023-10-23 185102](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/59647b47-b957-42ef-b6a4-78bad2831595)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Analoga a quella in classe
- Definiamo gli eventi $A=\\{a\ vince\ tutte\\}, B=\\{b\ vince\ tutte\\}, C=\\{c\ vince\ tutte\\}$
$\mathbb{P}(stesso\ cavallo\ vince\ tutte)=\mathbb{P}(A)+\mathbb{P}(B)+\mathbb{P}(C)$ perchè eventi disgiunti.
$\mathbb{P}(A)=\mathbb{P}(a\ vince\ 1°\cap a\ vince\ 2° \cap a\ vince\ 3°)= 0.3\cdot 0.3\cdot 0.3$ perchè le gare sono eventi indipendenti.
$\mathbb{P}(stesso\ cavallo\ vince\ tutte)=(0.3)^3+(0.5)^3+(0.2)^3=0.16$ 

- Analizziamo il caso in cui **a** vince la 1° gara, **b** la 2° e **c** la 3°: 
$\mathbb{P}(a\ vince\ 1°\cap b\ vince\ 2° \cap c\ vince\ 3°)=0.3\cdot 0.5\cdot 0.2$ siccome gli eventi (gare) sono indipendenti possiamo scrivere la probabilità dell'intersezione come prodotto delle probabilità dei singoli eventi.
I possibili abbinamenti di cavalli in modo che ognuno vinca una gara sono $3!=6$
Allora $\mathbb{P}(ogni\ cavallo\ ne\ vince\ una)= 6\cdot (0.2\cdot 0.5 \cdot 0.3)=0.18$

**Comment by daromacarlo** [comment]
![Immagine 2023-11-10 140514](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/0dca798a-3c61-443d-92f2-189fd2fb56c0)

**Comment by fraxb04** [comment]
> # Soluzione Analoga a quella in classe
> * Definiamo gli eventi A={a vince tutte},B={b vince tutte},C={c vince tutte}
>   P(stesso cavallo vince tutte)=P(A)+P(B)+P(C) perchè eventi disgiunti.
>   P(A)=P(a vince 1°∩a vince 2°∩a vince 3°)=0.3⋅0.3⋅0.3 perchè le gare sono eventi indipendenti.
>   P(stesso cavallo vince tutte)=(0.3)3+(0.5)3+(0.2)3=0.16
> * Analizziamo il caso in cui **a** vince la 1° gara, **b** la 2° e **c** la 3°:
>   P(a vince 1°∩b vince 2°∩c vince 3°)=0.3⋅0.5⋅0.2 siccome gli eventi (gare) sono indipendenti possiamo scrivere la probabilità dell'intersezione come prodotto delle probabilità dei singoli eventi.
>   I possibili abbinamenti di cavalli in modo che ognuno vinca una gara sono 3!=6
>   Allora P(ogni cavallo ne vince una)=6⋅(0.2⋅0.5⋅0.3)=0.18

non ho ben capito l'ultimo punto, perché moltiplichi lo spazio degli eventi con l'evento intersezione delle vincite?

**Comment by niccolo-fato** [comment]
![IMAGE 2024-02-15 19:12:17](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/73995334/0b81d0ff-eeea-42f7-9d13-29d99ca2fabd)
